controllers/propagator/propagation_test.go

```diff
@@ -233,3 +233,97 @@ func TestHandleDecisionWrapper(t *testing.T) {
 		close(resultsChan)
 	}
 }
+
+func (r MockPolicyReconciler) deletePolicy(
+	instance *policiesv1.Policy,
+) error {
+	return r.Err
+}
+
+func TestPlcDeletionWrapper(t *testing.T) {
+	// Simulate running Initialize. This is required for the retry library to actually run
+	// the handleDecision method.
+	attempts = 1
```

Comment:
I don't believe the retry library is used in your code. Could you try remove this part of the test?